addons/oceanbase-ce/dataprotection/restore.sh

```diff
@@ -1,112 +1,61 @@
+#!/usr/bin/env bash
 export PATH="$PATH:$DP_DATASAFED_BIN_PATH"
 export DATASAFED_BACKEND_BASE_PATH=${DP_BACKUP_BASE_PATH}
-sql_port_file=/home/admin/workdir/sql_port.ob
-sql_port=2881
-if [[ -f ${sql_port_file} ]];then
-  sql_port=$(cat ${sql_port_file})
-fi
-mysql_cmd="mysql -u root -h ${DP_DB_HOST} -P${sql_port} -N -e"
-OlD_IFS=$IFS
 
-archiveStatusFile="tenantStatus.dp"
-provider=
-access_key_id=
-secret_access_key=
-region=
-endpoint=
-bucket=
+SUBDOMAIN=${KB_CLUSTER_COMP_NAME}-headless
```

Comment:
remove the builti-in `KB_CLUSTER_COMP_NAME` env

Reply:
fixed